src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -637,11 +645,32 @@ async def get_requirements(
     # then it's owned by an exported target ET, and so R will be in the requirements for ET, and we
     # will require ET.
     direct_deps_tgts = await MultiGet(
-        Get(Targets, DependenciesRequest(tgt.get(Dependencies))) for tgt in owned_by_us
+        Get(
+            Targets,
+            DependenciesRequest(tgt.get(Dependencies), include_special_cased_deps=True),
+        )
+        for tgt in owned_by_us
     )
+
+    transitive_excludes: FrozenOrderedSet[Target] = FrozenOrderedSet()
+
```

Comment:
You can remove this white space so that the relevant code is better grouped together.